google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -307,6 +317,28 @@ public boolean getUseMultiplexedSession() {
     return useMultiplexedSession;
   }
 
+  @VisibleForTesting
+  @InternalApi
+  public boolean getUseMultiplexedSessionForBlindWrite() {
+    return useMultiplexedSessionForBlindWrite;
+  }
+
+  private static Boolean getUseMultiplexedSessionBlindWriteFromEnvVariable() {
```

Comment:
If we choose to keep this as a separate setting; Can we re-use the logic for getting a env-var that __must__ be either `true` or `false`?

Reply:
Have removed this funtion.